rails-questions.md

```diff
@@ -2,29 +2,40 @@
 
 1. What is SQL?
 
+Structured Query Language or SQL is a domain specific language used to manage data inside relational databases.
 
 2. What the PostgreSQL query that would return all the content in a particular table?
 
+SELECT * FROM table_name
 
 3. What is the command to create a new Rails application with a PostgreSQL database?
 
+$ rails new app_name -d postgresql -T
 
 4. What is the command to generate a Rails model for a meals table with columns named breakfast, lunch, and dinner?
 
+$ rails g model Meal breakfast:string lunch:string dinner:string
 
 5. What is a migration? Why would you use one?
 
+A migration is a file used to alter the database schema through Rails. Instead of writing SQL queries directly to manipulate the database, the migration file runs those queries for us. Making these independent helps to maintain the integrity of the database and gives us a history of what was added, changed, or removed.
```

Comment:
Yea, this also adds a layer of protection agains sql injection attacks. theres a funny XKCD comic about this

https://xkcd.com/327/